spec/spec_helper.rb

```diff
@@ -6,6 +6,7 @@
 require 'bundler/setup'
 Bundler.require(:default, :development)
 require 'soapy_cake'
+require 'webmock/rspec'
```

Comment:
It doesn't seem like you added anything that uses webmock. Is it to make the dependency explicit?